screens/JoinGroup.js

```diff
@@ -46,6 +47,24 @@ class JoinGroup extends Component {
     })
     //TODO: Add person to group if it matches, add group/habit to person profile'
     console.log('group ID: ',this.state.newGroupID)
+    const userId = firebase.auth().currentUser ? firebase.auth().currentUser.uid : "testAdminId"
+    const groupID = this.state.newGroupID;
+    const db = firebase.database().ref();
+    const moment = require('moment')
+    const today = moment().format('YYYY/MM/DD').split('/').join('')
+    const dateDict = {}
+    dateDict[today] = 0
+    db.child('/groups/'+groupID +'/groupMemberIds/'+userId).once("value")
+    .then(snapshot => {
+      if(!snapshot.val()) {
+        // create a userId reference to a list of dates
+        db.child('/groups/'+groupID+'/groupMemberIds/'+ userId).set(dateDict);
+      }
+    }).then(
+      this.props.navigation.navigate('Dashboard')
+    )
```

Comment:
We only want to join the group if the groupId is valid. Else, we don't do anything in the backend. I think this block of code should be in the 
if(groupIds.includes(this.state.newGroupID)){
        .... <- here
        this.joinGroupSuccessfulAlert()
 }